plugins/karate/Makefile.mk

```diff
@@ -1,8 +1,14 @@
 # LIBRARIES
 ##################################################
 if USE_KARATE
+# Plugin description is generated from README.md
+BUILT_SOURCES += plugins/karate/PluginDescription.h
+plugins/karate/PluginDescription.h: plugins/karate/README.md plugins/karate/Makefile.mk plugins/convert_README_to_header.sh
+	sh $(top_srcdir)/plugins/convert_README_to_header.sh plugins/karate PluginDescription.h
```

Comment:
Can you change the paths to similar to this:
https://github.com/OpenLightingProject/ola/blob/master/include/ola/Makefile.mk#L48

So you'll need to revert part of your most recent change, to pass in the paths separately.

I'm rather surprised it's not breaking Travis TBH.